### ISSUE TITLE
IMvxLanguageBinder.GetText still throws MvxException: Missing text provider

### ISSUE THREAD

**Issue by Krusty93** [MvvmCross/MvvmCross#4227]
## 🐛 Bug Report

When trying to get a text from a .rex file, Mvx throws an error:
`MvxException: Missing text provider - please initialize IoC with a suitable IMvxTextProvider`.
Browsing the source code I can see that `MvxLanguageBinder.GetText()` calls internally `Mvx.IoCProvider.TryResolve(out IMvxTextProvider cachedTextProvider)` and throws that exception when `IMvxTextProvider` is not properly registered into the IoC container.

However, my app can resolve `IMvxTextProvider`. Details in the snippet below:
```
var isResolved = Mvx.IoCProvider.TryResolve(out IMvxTextProvider textProvider); // isResolved is true and textProvider is not null
var message = LocalizedTextSource.GetText("RestoreAccountMessage"); // throws the exception
```

The base view model implements `IMvxLocalizedTextSourceOwner`.
The plugin is registered in `MvxApplication.Startup`, but moving it to `Initialize` or somewhere else does not fix the issue.

Note: I have this issue since Mvx 8.0.1. Downgrading to Mvx7 fixes the issue

### Expected behavior
The snipper above doesn't throw any exception and the localized text is obtained

### Reproduction steps
- Install MvvmCross.Plugin.ResxLocalization 8.0.1 in Core and platform specific project
- Initialize it adding `Mvx.IoCProvider.RegisterSingleton<IMvxTextProvider>(new MvxResxTextProvider(AppResources.ResourceManager));` in `MvxApplication.Startup`
- Make your view model implements `IMvxLocalizedTextSourceOwner`
- Call `IMvxLanguageBinder.GetText("");`

**Tested on Android only**

### Configuration

**Version:** 8.0.1

**Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by asoodz** [comment]
I have the same issue, any update on this, will be very helpful.

Thanks

**Comment by Cheesebaron** [comment]
"I have the same issue, share ETA for fix please" comments are not useful. Comment deleted.

**Comment by Cheesebaron** [comment]
Duplicate of #4208 fixed in #4209
